openlibrary/tests/core/test_processors.py

```diff
@@ -28,7 +28,7 @@ def _request(self, path, method=None, data=None):
             olid = data['olid']
             return web.storage(key=self.olids.get(olid))
 
-    def _get_backreferences(self):
+    def _get_backreferences(self, *args, **kwargs):
```

Comment:
this technically fixes a different booklending_utils bug which makes tests fail on dev (because the booklending_utils lib is actually present)